src/lib/viewers/controls/annotations/AnnotationsTargetedTooltip.tsx

```diff
@@ -1,52 +0,0 @@
-import React from 'react';
-import TargetedClickThroughTooltip from '../tooltip';
-import { ControlsLayerContext } from '../controls-layer';
-import { ExperiencesContext } from '../experiences';
-import { TargetingApi } from '../../../types';
-import './AnnotationsTargetedTooltip.scss';
-
-export type Props = React.PropsWithChildren<{
-    isEnabled?: boolean;
-}>;
-
-function AnnotationsTargetedTooltip({ children, isEnabled = false }: Props): JSX.Element | null {
```

Comment:
We could do without changing it, but I feel it's better to keep it as it is. When this component was defined there was only a single tooltip on Annotations. Now there are 2. IMO it makes sense to:
- Rename it, as not it's not the only Annotations-targeted tooltip anymore, so it's old name is misleading
- As now there are 2 tooltips, I created a wrapper `AnnotationsButtonRegionTooltips` component to switch between them. `AnnotationsTargetedTooltip` is used only locally by that new wrapper component. I moved it to the directory of that component because of that.